### PR TITLE
chore: Export JWS functions

### PR DESCRIPTION
**Pull request by oed**
Forgot to add these exports in my last PR.

FYI, looks like tests are passing but two tests are broken and not being awaited on develop.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import SimpleSigner from './SimpleSigner'
 import NaclSigner from './NaclSigner'
-import { verifyJWT, createJWT, decodeJWT, Signer } from './JWT'
+import { verifyJWT, createJWT, decodeJWT, verifyJWS, createJWS Signer } from './JWT'
```
Comment (mirceanis) [inline src/index.ts:3]: This is missing a comma
```suggestion
import { verifyJWT, createJWT, decodeJWT, verifyJWS, createJWS, Signer } from './JWT'
```

**Comment by mirceanis** [comment]
I changed the default branch to `master`; the false positive tests from `develop` were already fixed in #88

File: src/index.ts (hunk 1 above)
Comment (oed) [reply src/index.ts:3]: 🤦

**Comment by oed** [comment]
@mirceanis should I rebase this on master then?

**Comment by mirceanis** [comment]
yes, please do

**Pull request merged**